ansible/roles/products/teleop/server/desktop-icons/templates/scripts/wait_for_topic.j2

```diff
@@ -0,0 +1,17 @@
+#jinja2: trim_blocks:False
+#!/bin/bash
+
+# Arg1: Topic name to wait for
+# Arg2: Error message to display if topic doesn't appear until timeout
+elapsed_time=0
+start_time=$SECONDS
+while [ $elapsed_time -lt 60 ]; do
```

Comment:
could we make the "60" a param just for flexibility sake? Or at least move it to a "const" variable?

Reply:
Extracted constant in e7efe2ff5007e7e4596c4c507386c023cd712690